api-model/src/converter.rs

```diff
@@ -274,3 +280,33 @@ impl CertificateConfig {
         }
     }
 }
+
+/// Describes the information required to access CCM
+#[derive(Clone, Eq, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct CcmConfiguration {
+    pub ccm_url: String,
+}
+
+impl Default for CcmConfiguration {
+    fn default() -> Self {
+        CcmConfiguration {
+            ccm_url: "ccm.fortanix.com:443".to_string(),
+        }
+    }
+}
+
+/// Describes the information required to access DSM
+#[derive(Clone, Eq, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct DsmConfiguration {
+    pub dsm_url: String,
```

Comment:
Can you make this private?

Reply:
I can't. It is used in the converter crate to check if it's a valid url.